src/masternode-sync.h

```diff
@@ -13,7 +13,7 @@ class CMasternodeSync;
 
 static const int MASTERNODE_SYNC_FAILED          = -1;
```

Comment:
Give these constants more useful names and types:

- The first 9 MASTERNODE_SYNC_* constants appear to be an ordered list of stages of synchronization<br>(but are referred to by the `int nRequestedMasternodeAssets` private member).<br>(The absolute values don't appear to be important, just the ordering.)
- The next 2 MASTERNODE_SYNC_*_SECONDS are time durations.
- The final MASTERNODE_SYNC_ENOUGH_PEERS is just a constant int.

To the reader it is hard to see the difference - all have similar names and identical declared types.

**Naming**
MASTERNODE_SYNC is just a 'namespace' prefix adding verbosity without giving information or context.
Cryptic names IBD and MNW should be expanded or explained in comments.
How is INITIAL different from IBD.
How is GOVERNANCE different from GOVOBJ.

**Types**
All have the same type `const int`. They are three different types of constant. Appropriate types give the reader and the compiler more information.

**Scope / linkage**
The internal linkage reduces the need for prefixing, as the scope is limited to the translation unit.
Anonymous namespace is preferred over the `static` keyword for specifying internal linkage.

**Suggestion**
```
#include <chrono>

namespace
{
    enum class SyncStage  // AssetSyncStage? 
    {
        FAILED,
        INITIAL,    // ?
        IBD,        // Initial Block Download
        LIST,
        GOVERNANCE,
        GOVOBJ,
        GOVOBJ_VOTE,
        FINISHED
    };
    
    std::chrono::seconds const TICK{6};
    std::chrono::seconds const TIMEOUT{30};
    
    constexpr int ENOUGH_PEERS = 6;
}

```